src/i18n/fr/nav.ts

```diff
@@ -21,32 +25,33 @@ export default NavDictionary({
 	'core-concepts/astro-components': 'Composants',
 	'core-concepts/astro-pages': 'Pages',
 	'core-concepts/layouts': 'Layouts',
-	'guides/markdown-content': 'Markdown',
+	'guides/markdown-content': 'Contenu Markdown',
 	'guides/imports': 'Fichiers Statiques',
 	'guides/troubleshooting': 'Dépannage',
 
 	// Features
 	features: 'Fonctionnalités',
 	'guides/configuring-astro': 'Configurer Astro',
-	'guides/styling': 'CSS et Styles',
-	'guides/data-fetching': 'Récupération de Données',
+	'guides/styling': 'CSS et Style',
+	'guides/data-fetching': 'Récupération de données',
 	'guides/deploy': 'Déployer',
 	'guides/environment-variables': "Variables d'Environnement",
-	'guides/aliases': "Alias d'Importation",
+	'guides/aliases': 'Alias d’importation',
 	'guides/integrations-guide': 'Intégrations',
 	'guides/rss': 'Flux RSS',
 	'guides/server-side-rendering': 'Rendu côté serveur (SSR)',
 	'guides/typescript': 'TypeScript',
 	'core-concepts/framework-components': 'Composants de Frameworks',
 
-	// Reference
+	// Référence
 	reference: 'Référence',
 	'reference/configuration-reference': 'Configuration',
 	'reference/cli-reference': 'Lignes de Commandes',
-	'reference/api-reference': 'API d\'Exécution',
-	'reference/integrations-reference': 'API d\'Intégration',
+	'reference/api-reference': "API d'Exécution",
+	'reference/integrations-reference': "API d'Intégration",
 	'reference/adapter-reference': 'API des adaptateurs (expérimental)',
 	'core-concepts/routing': 'Règles de Routage',
 	'reference/directives-reference': 'Utilisation des Directives',
-	'guides/publish-to-npm': 'Format de Packet NPM',
-});
+	'guides/publish-to-npm': 'Publier sur NPM',
```

Comment:
In English we have “NPM Package Format” as the label, so I think the original French translation here was correct.

```suggestion
	'guides/publish-to-npm': 'Format de Packet NPM',
```